kolibri/plugins/learn/assets/src/routes/index.js

```diff
@@ -99,23 +89,23 @@ export default [
     },
   },
   {
-    name: PageNames.TOPICS_CHANNEL,
-    path: '/topics/:channel_id',
```

Comment:
This feels like a notable change. Not blocking, and not a technical comment, but I'm wondering if this might require some toolkit or other updates for public channels to places like on the demo server, etc? I'm not sure how many places we have links to specific channel resources but thought that might be worth flagging

Reply:
I think it would probably be a good idea to retain the route, and instead do a redirect to the new page, in the interests of the above.

Reply:
Have added the redirect.